clients/gitlabrepo/client.go

```diff
@@ -236,6 +236,11 @@ func (client *Client) GetCreatedAt() (time.Time, error) {
 	return client.project.getCreatedAt()
 }
 
+func (c *Client) GetMFARequired() (required bool, err error) {
```

Comment:
Appears this is available on GitLab for group namespaces as well:
https://docs.gitlab.com/ee/api/groups.html